edb/common/debug.py

```diff
@@ -171,6 +171,11 @@ def dump_code(*args, **kwargs):
     _markup.dump_code(*args, **kwargs)
 
 
+def dump_sql(sql, *args, **kwargs):
```

Comment:
nit: Perhaps no need for `*args` and `**kwargs`

Reply:
Aha, those are supposed to be forwarded to `dump_code`. Which, also, to be honest, there is probably no need for, but I figured I'd be consistent.